src/recon.rs

```diff
@@ -1796,9 +1800,11 @@ unsafe fn read_coef_tree<BD: BitDepth>(
         let mut cbi_idx = 0;
         if (*t).frame_thread.pass != 0 {
             let p = (*t).frame_thread.pass & 1;
-            cf = CfSelect::Frame(ts.frame_thread[p as usize].cf);
-            ts.frame_thread[p as usize].cf +=
-                cmp::min((*t_dim).w, 8) as usize * cmp::min((*t_dim).h, 8) as usize * 16;
+            cf = CfSelect::Frame(ts.frame_thread[p as usize].cf.load(Ordering::Relaxed));
+            ts.frame_thread[p as usize].cf.fetch_add(
+                cmp::min((*t_dim).w, 8) as usize * cmp::min((*t_dim).h, 8) as usize * 16,
+                Ordering::Relaxed,
+            );
```

Comment:
We can just do a `load`, a normal add, and then a `store`, right?  I don't think we need `fetch_add` (and I'm not sure if it's zero overhead like relaxed loads and stores are).  Same for anywhere else a similar pattern happens, though some other places you do it this way already.